public/js/components/modal.jsx

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+var React = require('react');
+var gettext = require('utils').gettext;
+var cx = require('classnames');
+
+
+module.exports = React.createClass({
+
+  displayName: 'Modal',
+
+  propTypes: {
+    children: React.PropTypes.array.required,
```

Comment:
bah, this might not be possible http://stackoverflow.com/questions/27006782/force-reactjs-to-throw-real-errors-when-proptypes-validation-fails